content/articles/print-is-device.md

```diff
@@ -0,0 +1,178 @@
+
+In this post, I offer some guidance, based on my experience, because indeed printing from web to paper is not as straightforward as one might assume.
+
+## Print is a device
+
+The printed paper is a portable, static reading device. It is a component of adaptive design alongside desktop and mobile devices, and assistive technology for accessibility. Because of this, it should be given ample consideration in the web development process.
+
+What you see on the screen is not necessarily what you’ll get when you print because the printed version is not a facsimile of the webpage. The printer renders HTML, applies CSS, and then transfers ink onto the output: physical, fixed size pages of paper. Unless specified exclusively for screen use, all defined general CSS rules, even those for responsive CSS, are applied on the printed output. Therefore, you might see some unintended display on paper if the rule in the general CSS is not filtered out. The print stylesheet rules are needed to accommodate the differences between screen and print.
+
+{{< figure src="/img/assets/print-css.png" class="text-center centered responsive" alt="" >}}
+
+The objective is to design and develop well for print, not to recreate a screenshot of a webpage.
+
+## Print should be part of the process
+
+Printing is like a pizza with toppings—you can’t simply put pepperoni, or vegetables, whichever toppings of your choice on the pizza after it’s done baking. The toppings need to be one with the cheese and sauce, in harmony.
+
+Similar to mobile and accessibility design, print cannot be attached to the web development process as an afterthought, or eliminated as a result of time constraints. Because of its static nature, there is a complementary UA and UI for the printed version. For example, delivery of multimedia and interactive content is completely lost in print. Thus, alternative options have to be offered; otherwise, critical content may be missing or distorted.
+
+I recommend a thoughtful design for print as a corresponding piece for the screen display. It’s also a good practice to prepare and test the layout, architecture, and “behavior” of the webpage elements as you’re designing for digital devices.
+
+## Decide what is fit to print
+
+You can’t print the active and dynamic nature of the internet: there is no multimedia, no hyperlink, no navigational tools, and no interactivity on paper. Cues such as “click here” or “view the clip” are meaningless when printed.
+
+In the web development process, consider what the users of your website will take with them when they print. The following are a few things to be aware of:
+
+### Readability:
+For the printed pages, the conditions how the text is read are different. Black text on white background provides the maximum contrast suitable for paper, especially if a user decides to print in black and white. Light text on dark background should be avoided.
```

Comment:
*For the printed pages, the conditions how the text is read*: Check your plurals here.